src/sentry/apidocs/examples/notification_examples.py

```diff
@@ -72,6 +72,78 @@
     )
 ]
 
+GET_NOTIFICATION_ACTION = [
+    OpenApiExample(
+        "Retreive a spike protection notification action created for the the project, The Spoiled Yoghurt",
+        value={
+            "id": "836501735",
+            "organizationId": "62848264",
+            "serviceType": "sentry_notification",
+            "targetDisplay": "default",
+            "targeyIdentifier": "default",
+            "targetType": "specific",
+            "triggerType": "spike-protection",
+            "projects": [
```

Comment:
the projects are actually returned as a list of IDs

Reply:
🤦‍♀️  I just merged the other ones (list and create) like this. Let me fix them all

Reply:
oh oops